earth_enterprise/rpms/buildSrc/src/main/groovy/GeeDeb.groovy

```diff
@@ -82,4 +96,19 @@ class GeeDeb extends com.netflix.gradle.plugins.deb.Deb {
 
         findRequires(packageInputFiles).each { requires(it) }
     }
-}
\ No newline at end of file
+
+    protected void formatPackageDescription(String descrip){
+        packageDescription = ''
+        List lines = descrip.split( '\n' )
+        lines.eachWithIndex{ item, index ->
```

Comment:
Minor: I think you could do with:

```Groovy
descrip.eachLine { line ->
    // . . .
}
```

Reply:
Very handy. I didn't know about eachLine. Thanks.